go.mod

```diff
@@ -22,6 +22,7 @@ require (
 	github.com/mattn/go-isatty v0.0.16
 	github.com/mongodb-forks/digest v1.0.4
 	github.com/mongodb-labs/cobra2snooty v0.12.2
+	github.com/mongodb/mongodb-atlas-kubernetes v1.3.1-0.20221024110443-0c2d71e4f1e4
```

Comment:
could we use a real version and not a pseudo/unreleased tag?

Reply:
Updated to the latest release 1.4.1